install/terraform/modules/gke-autopilot/cluster.tf

```diff
@@ -63,6 +65,21 @@ resource "google_container_cluster" "primary" {
   }
   min_master_version = local.kubernetesVersion
 
+  maintenance_policy {
+    # When exclusions and maintenance windows overlap, exclusions have precedence.
+    daily_maintenance_window {
+      start_time = "03:00"
+    } 
+    maintenance_exclusion{
+      exclusion_name = format("%s-%s", local.name, "exclusion")
+      start_time = local.maintenanceExclusionStartTime
+      end_time = local.maintenanceExclusionEndTime
+      exclusion_options {
+        scope = "NO_MINOR_OR_NODE_UPGRADES"
```

Comment:
I think we're ok with just `NO_MINOR_UPGRADES`. On GKE the nodes will never get upgraded past the minor version of the master, so `NO_MINOR_UPGRADES` locks the nodes, too - but will allow the version to be updated _within_ the minor.